templates/openstackplaybookgenerator/bin/create-playbooks.sh

```diff
@@ -38,17 +38,17 @@ rm -Rf "$TEMPLATES_DIR"
 mkdir -p $TEMPLATES_DIR
 
 cp -a /usr/share/openstack-tripleo-heat-templates/* $TEMPLATES_DIR
-pushd $TEMPLATES_DIR
-# extract any tar files into the $TEMPLATES_DIR
+
+# copy to editable dir config-tmp
+rm -Rf $HOME/config-tmp
```

Comment:
Why do you need to remove it? Shouldn't it always not exist?

Reply:
as discussed on slack we should keep that when running in interactive mode you want to rerun the script